src/Metadata/Property/PropertyMetadata.php

```diff
@@ -43,6 +44,7 @@ public function __construct(Type $type = null, string $description = null, bool
         $this->identifier = $identifier;
         $this->iri = $iri;
         $this->attributes = $attributes;
+        $this->childInherited = null;
```

Comment:
It's not mandatory to use factories. For a small API you may want to instantiate metadata VO directly.